### PR TITLE
Update the Containers FAQ Doc

### PR DESCRIPTION
**Pull request by TSmith1992**
@kennonkwok @ceciliawatt 

### What does this PR do?

This PR will revamp the Containers Troubleshooting page and provide more tips, as well as what the Support Team will generally request upon initial investigation. 

### Motivation

As part of my embed program, I have been tasked with revamping our Containers Docs. This PR is inspired by the APM Troubleshooting doc (https://docs.datadoghq.com/tracing/troubleshooting/#pagetitle)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by hestonhoffman** [comment]
Hi, I added a [docs editorial card](https://datadoghq.atlassian.net/browse/DOCS-5085) for this one. We'll assign someone to take a look soon

**Review by TSmith1992 — COMMENTED** [review]
Changes requested appear very minimal/cosmetic. I approve them. Thank you

File: content/en/containers/troubleshooting/_index.md [hunk 1]
```diff
@@ -1,9 +1,210 @@
...
+
+ - There are no proxies or firewalls that might impede the Agent from accessing the endpoint. 
+
+ - Agent has [Autodiscovery](https://docs.datadoghq.com/getting_started/containers/autodiscovery/?tab=adannotationsv2agent736#overview) enabled.
+
+ 
+
+**Not getting logs:**
+
+There are two [environment variables](https://docs.datadoghq.com/agent/guide/environment-variables/#overview) that can impact if logs will be collected and, if so, from which containers. The first variable is `DD_LOGS_ENABLED`, which must be enabled. To collect all logs from all containers, you can set the environment variable `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL` to `true`.
```
Comment (TSmith1992) [inline content/en/containers/troubleshooting/_index.md:49]: There are two [environment variables](https://docs.datadoghq.com/agent/guide/environment-variables/#overview) that can impact if logs can be collected and, if so, from which containers. The first variable is `DD_LOGS_ENABLED`, which must be enabled. To collect all logs from all containers, you can set the environment variable `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL` to `true`.

File: content/en/containers/troubleshooting/_index.md [hunk 2]
```diff
@@ -1,9 +1,210 @@
...
+
+To exclude logs (and other features) from collection, please refer to Datadog’s [Container Discovery Management page](https://docs.datadoghq.com/agent/guide/autodiscovery-management/?tab=containerizedagent#pagetitle) for more insight.
```
Comment (TSmith1992) [inline content/en/containers/troubleshooting/_index.md:51]: To exclude logs (and other features) from collection, see Datadog’s [Container Discovery Management page](https://docs.datadoghq.com/agent/guide/autodiscovery-management/?tab=containerizedagent#pagetitle) for more insight.

File: content/en/containers/troubleshooting/_index.md [hunk 3]
```diff
@@ -1,9 +1,210 @@
...
+ - Set `TLS_VERIFY` to `false`.
+
+
+ 
+
+**HPA metrics are not appearing or are not alining with the expected value.**
+
+- Ensure that the Cluster Agent is deployed and able to send data to the Node Agent
+
+- Review the query used to scale the external metrics in the Metrics Summary. Only valid queries will autoscale, and if there are multiple queries, ALL will be ignored if **ANY** of the queries are invalid.
```
Comment (TSmith1992) [inline content/en/containers/troubleshooting/_index.md:70]: - Review the query used to scale the external metrics in the Metrics Summary. Only valid queries may autoscale, and if there are multiple queries, ALL will be ignored if **ANY** of the queries are invalid.

**Review by TSmith1992 — COMMENTED** [review]
Made most requested style changes, save some cases where the sentence was deemed too long.

**Comment by cswatt** [comment]
For future reference (I totally should've remembered to tell you this!) branches must be named `<username>/<feature-name>` to pass the gitlab build. I made further edits to this PR and opened a new branch/PR: https://github.com/DataDog/documentation/pull/17475

Closing this one

**Pull request closed without merging**